pom.xml

```
@@ -40,36 +34,43 @@
     </dependency>
 
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
-      <version>1.7.25</version>
-      <scope>test</scope>
+      <groupId>com.github.sahabpardaz</groupId>
+      <artifactId>log-throttle</artifactId>
+      <version>v1.0</version>
     </dependency>
 
     <dependency>
-      <groupId>org.apache.kafka</groupId>
-      <artifactId>kafka_2.13</artifactId>
-      <version>${kafka.version}</version>
+      <groupId>com.github.sahabpardaz</groupId>
+      <artifactId>label-supported-dropwizard-metrics</artifactId>
+      <version>1.0.3-dropwizard4</version>
```

```suggestion
      <version>1.0.3</version>
```

Sepid does not support Dropwizard4.
We are going to solve it and remove the extra branch of label-supported-dropwizard-metrics.
Please wait until then

---

as we talked